test/e2e/antreapolicy_test.go

```diff
@@ -3686,81 +3687,101 @@ func testACNPICMPSupport(t *testing.T, data *TestData) {
 
 func testACNPNodePortServiceSupport(t *testing.T, data *TestData) {
 	skipIfProxyAllDisabled(t, data)
+	testEndpointOnDiffNode := false
+	currentEncapMode, err := data.GetEncapMode()
+	if err != nil {
+		t.Fatalf("Failed to get encap mode: %v", err)
+	}
+	if currentEncapMode == config.TrafficEncapModeEncap {
+		testEndpointOnDiffNode = true
+	}
```

Comment:
Is that ANP ingress for NodePort is only available in encap mode? Why don't test Endpoint on diff Node in noEncap?

Reply:
Changed. Previous I was confused by the reject packet_out in_port implementation.